example-templates/liquid-rhai-bridge/rhai-example.rhai

```diff
@@ -0,0 +1,5 @@
+print("Inside the rhai filter function");
+
+variable::set("some_liquid_variable", "You can even set values from within the rhai filter function");
```

Comment:
is the expectation that those variables set in rhai exists over the whole lifetime of cargo generate or only for this very rhai script?

Reply:
Definitely the whole - variables not designated as placeholders but assigned from `liquid` only last for the file in which they are defined, while these last from definition and onwards. This is consistent with the behavior if doing it in a hook function.

Come to think about it, it would be nice if we could somehow enforce the order of processing files. That would simplify some of the scripts a user might write (thinking of #847 here)